nUpdate.UpdateInstaller/WindowsServiceHelper.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace nUpdate.UpdateInstaller
+{
+    public static class WindowsServiceHelper
+    {
+        private static bool? _IsRunningInServiceContext;
+
+        /// <summary>
+        /// Indicates if the application runs in service context

```

Comment:
Please start the doc with "Gets a value indicating, if ..."

Reply:
done